src/debug.c

```diff
@@ -1724,11 +1724,14 @@ void sigsegvHandler(int sig, siginfo_t *info, void *secret) {
 
     bugReportStart();
     serverLog(LL_WARNING,
-        "Redis %s crashed by signal: %d", REDIS_VERSION, sig);
-    if (sig == SIGSEGV || sig == SIGBUS) {
+        "Redis %s crashed by signal: %d, si_code: %d", REDIS_VERSION, sig, info->si_code);
+    if (sig == SIGSEGV || sig == SIGBUS || sig == SIGFPE) {
```

Comment:
SIGFPE gives you the address of the failing command, unlike the other two signals which give you the address of the failing memory access. Since we already have the stack trace, I'm not sure this adds any value and might cause a little bit of confusion because of the wording of the error.

Reply:
makes sense, i'll drop it.